pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -329,14 +328,16 @@ def write_region(self, regional_input: RegionalInput) -> None:
                     schema.FIPS,
                 )
             ]
+            # Casing floats to ints and then replacing filled in zeros with NaN values instead of
+            # propagating zeros.
+            na_int_columns = output_model.loc[:, int_columns].isna()
             output_model.loc[:, int_columns] = output_model[int_columns].fillna(0).astype(int)
+            output_model[na_int_columns] = np.nan
```

Comment:
this looks like you are using a data frame (with int_columns columns) for bool indexing. Does that work? I don't see it document at https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#boolean-indexing